### ISSUE TITLE
No License

### ISSUE THREAD

**Issue by lbrigmanArris** [raffaelespazzoli/openshift-enablement-exam#17]
Without a License on the project, officially no one can use it except by explicit written permission from the copyright holder.

Please add a license to the project.

**Comment by raffaelespazzoli** [comment]
I am not an expert on this subject, would you be able to suggest an appropriate license?

**Comment by srgvg** [comment]
I would suggest choosing the same license as https://github.com/openshift/openshift-ansible, being the Apache License 2.0.

see https://github.com/openshift/openshift-ansible/blob/master/LICENSE

**Comment by lbrigmanArris** [comment]
I agree with srgvg's comment.  The apache 2.0 license as matching that of OpenShift-ansilble.

**Comment by raffaelespazzoli** [comment]
fixed via #18 
thanks @lbrigmanArris 
@srgvg do you agree that we can close this?

**Comment by srgvg** [comment]
Yes, this is perfect, thank you all!
